### DISCUSSION TITLE
How to deal with bugs caused by the rubocop core

### DISCUSSION THREAD

**Discussion by Darhazer** [rubocop-hq/rubocop-rspec#1264] [General]
Recently I've [fired a bug report](https://github.com/rubocop/rubocop/issues/10469) that `CountAsOne` can lead to negative counts. This one affects `RSpec/ExampleLength` - which is how I found about it in the first place (well, to be fair, a colleague of mine reported the issue, and I took it from there). 

Now that the issue is fixed, and it will be out with the next rubocop release, I have this general question: how we should proceed with it, as it essentially fixes our cops as well.

Should we bump the minimum required version of rubocop, to ensure it counts statements correctly, and release rubocop-rspec with a "fix" for that option?
Should we provide a message that CountAsOne has buggy behaviour for rubocop versions prior to X?
Should we do something else?

**Comment by pirj** [comment]
> bump the minimum required version of rubocop, to ensure it counts statements correctly, and release rubocop-rspec with a "fix" for that option

Without doing such a release, `bundle update rubocop-rspec` without the `--conservative` flag would trigger the `rubocop` update, and the bug would be fixed anyway?
For new projects, the latest version of `rubocop` with the fix would be bundled as well.

**Comment by pirj** [comment]
I think we're safe with the "min" and "edge" RuboCop CI jobs.
